cpp/inc/bond/core/detail/double_pass.h

```diff
@@ -3,7 +3,8 @@
 
 #pragma once
 
-#include <bond/core/config.h>
```

Comment:
If config.h is included, is there a problem? If so, can that be fixed instead?

The [Bond C++ coding guidelines](https://github.com/microsoft/bond/blob/master/cpp/CODING_GUIDELINES_CPP.md#header-file-includes) requires that every file include config.h

Reply:
No problem, it's just that in last PR Ara was asking to remove `config.h` because `bond_fwd.h` already includes it :). Will restore, thanks!

Reply:
Fixed